package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -54,11 +58,35 @@ def _bootstrap(project_path: Path):
         return
 
 
+def get_dataset_stats(project_path: Path) -> Dict:
+    """Return the stats saved at stats.json as a dictionary if found.
+    If not, return an empty dictionary
+
+    Args:
+        project_path: the path where the Kedro project is located.
+    """
+    try:
+        stats_file_path = project_path / "stats.json"
+
+        if not stats_file_path.exists():
+            return {}
+
+        with open(stats_file_path, encoding="utf8") as stats_file:
+            stats = json_lib.load(stats_file)
+            return stats
+
+    except Exception as exc:  # pylint: disable=broad-exception-caught
+        logger.warning(
+            "Unable to get dataset stats from project path %s : %s", project_path, exc
```

Comment:
Since this error can appear in logs for users I think it's better to not use abbreviations. 

```suggestion
            "Unable to get dataset statistics from project path %s : %s", project_path, exc
```